runtime/doc/lua.txt

```diff
@@ -913,6 +913,54 @@ vim.fn.{func}({...})					*vim.fn*
         Note: vim.fn keys are generated lazily, thus `pairs(vim.fn)` only
         enumerates functions that were called at least once.
 
+vim.define_augroup({group_name}, {clear_autocmds})       *vim.define_augroup()*
+        Defines a new |augroup| with name {group_name}. If {clear_autocmds} is
+        true, then it will clear all the autocmds currently in {group_name}.
+
+        Parameters: ~
+            {group_name}: |augroup| name.
+            {clear_autocmds}: If `true`, clears existing autocmds.
```

Comment:
Maybe a separate `vim.clear_augroup` command would be useful (instead or in addition), as an equivalent to vim's `au!`? I can see this being useful enough to merit being available directly.

(Ideally, we would replicate the full api and offer equivalents to `augroup!` and `doautocommand` as well, but those might be more niche and acceptable to call indirectly.)

Reply:
Yeah, I can add this. It seems good. I will expose programmatic way to interact with autocommands, rather than just through vim.cmd.

I will make vimscript APIs for them as well I think.